Source/ACE.Server/WorldObjects/Chest.cs

```diff
@@ -55,7 +55,9 @@ private void SetEphemeralValues()
             //Eventually these case statements would be linked to indivual treasure generators. Each one should be a different profile, but currently it will be the complete appropriate tier profile.
             for (int i = 0; i < GeneratorProfiles.Count; i++)
```

Comment:
in the generator profiles.. if the where create hasflag RegenLocationType.Treasure
it should do a look up in the death treasure table for that value of the wcid
if its just Contain, it needs no look up and that's the actual wcid

right now the code below is assuming anything below 500 is a death treasure which could be wrong